Floofbot/Floofbot.csproj

```diff
@@ -5,6 +5,11 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Migrations\20200501113559_AddNicknameAlert.cs" />
```

Comment:
What's this part doing?

Reply:
It deletes the migration file after the migration has been added to the database, not compulsory